### PR TITLE
Fix outdated mirror, Use official centOS 7.0 base image

### PR DESCRIPTION
**Pull request by niyatpatel23295**
Outdated mirror causing following error on build.
```
 => ERROR [ 2/37] RUN yum install -y curl which tar sudo openssh-server openssh-clients rsync yum-plugin-ovl    && yum update -y libselinux                                                                1.9s
------
 > [ 2/37] RUN yum install -y curl which tar sudo openssh-server openssh-clients rsync yum-plugin-ovl    && yum update -y libselinux:
#5 0.314 Loaded plugins: fastestmirror, keys, protect-packages, protectbase
#5 0.318 Setting up Install Process
#5 0.346 Determining fastest mirrors
#5 1.823 Error: Cannot find a valid baseurl for repo: base
#5 1.823 YumRepo Error: All mirror URLs are not using ftp, http[s] or file.
#5 1.823  Eg. Invalid release/repo/arch combination/
#5 1.823 removing mirrorlist with no valid mirrors: /var/cache/yum/x86_64/6/base/mirrorlist.txt
```

Fixes #XXXX.

#### Added a forbidden-apis entry ...

<hr>

##### Key changed/added classes in this PR
 * `MyFoo`
 * `OurBar`
 * `TheirBaz`

<hr>

This PR has:
- [ ] been self-reviewed.
   - [ ] using the [concurrency checklist](https://github.com/apache/druid/blob/master/dev/code-review/concurrency.md) (Remove this item if the PR doesn't have any relation to concurrency.)
- [ ] added documentation for new or modified features or behaviors.
- [ ] added Javadocs for most classes and all non-trivial methods. Linked related entities via Javadoc links.
- [ ] added or updated version, license, or notice information in [licenses.yaml](https://github.com/apache/druid/blob/master/dev/license.md)
- [ ] added comments explaining the "why" and the intent of the code wherever would not be obvious for an unfamiliar reader.
- [ ] added unit tests or modified existing tests to cover new code paths, ensuring the threshold for [code coverage](https://github.com/apache/druid/blob/master/dev/code-review/code-coverage.md) is met.
- [ ] added integration tests.
- [ ] been tested in a test Druid cluster.

### CODE REVIEW

File: examples/quickstart/tutorial/hadoop/docker/Dockerfile [hunk 1]
```diff
@@ -14,14 +14,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Creates pseudo distributed hadoop 2.8.3 with java 8
-#
-# Modified from the SequenceIQ Dockerfiles at https://github.com/sequenceiq/hadoop-docker
-#
-# docker build -t druid-hadoop-demo:2.8.3 .
```
Comment (xvrl) [inline examples/quickstart/tutorial/hadoop/docker/Dockerfile:21]: for licensing reasons we should leave the code attribution to credit the source of this code

**Comment by xvrl** [comment]
@niyatpatel23295 please retarget your PR to the master branch instead of an old version

**Comment by clintropolis** [comment]
I think this has already been done with #10638.

**Comment by xvrl** [comment]
closing since this has already been addressed

**Pull request closed without merging**